install-storageos/070-cluster-licence.md

```diff
@@ -0,0 +1,7 @@
+StorageOS clusters need a licence to operate after 24 hours. However, creating a developer licence is easy to do and completely free!
+
+To create a licence click on the StorageOS UI Licence tab of the terminal and click on the `Sign Up` button under the Developer Licence box. Follow the instruction and congratulations, you now have a working Licenced StorageOS cluster. 
```

Comment:
Make sure we call out that after account creation 'don't forget to activate the account in your email' - because without that step the user can't request and apply the license and it isn't obvious in the GUI.